test/gtest/common/test.h

```diff
@@ -25,6 +25,8 @@
 #include <vector>
 #include <string>
 
+#define UCC_CHECK(_call)    EXPECT_EQ(UCC_OK, (_call))
```

Comment:
i think this one should be ASSERT_EQ. If ucc call fails (e.g. collective_init, or coll_test) there is no sense to trying to proceed with the data check - test should be aborted.

Reply:
fixed